opencti-platform/opencti-graphql/src/migrations/1724143733795-trigger-authorized-authorities.js

```diff
@@ -0,0 +1,34 @@
+import { logApp } from '../config/conf';
```

Comment:
Could you rebase master and update the timestamp of this migration please ?

Reply:
Done